test/dynamo/test_functions.py

```diff
@@ -281,6 +283,9 @@ def test_load_global_bool(x):
             return x
 
     @make_test
+    @patch.object(
+        torch._dynamo.config, "dynamic_shapes", False
+    )  # TypeError: 'torch._C.SymIntNode' object cannot be interpreted as an integer
```

Comment:
> I find these patch calls puzzling because afaict you did not actually change the default config setting. Could it be that you are trying to patch the config for when these tests are dupes for dynamic shapes? If so, it would be better to xfail them manually in the dynamic shapes test file. There's an example of this in the suppress_errors PR that you reviewed...

All tests are run multiple times with different config settings. The two major combinations are dynamic shapes on/off and take tensor on / off.

Reply:
OK, that's what I thought. But then patching it in the way you've done here means we *still* run the test twice (once without dynamic shapes, once with dynamic shapes), except sekritly the dynamic shape run isn't actually a dynamic shape run.

It would be better to xfail like I did in https://github.com/pytorch/pytorch/pull/87440/files#diff-f0c247e8e8b2d888c57629af4e2747d7f981f4cfee0d5ee7095058db1db9c9c7R28 and then you also get to find out when the test stopped failing.

Reply:
Sure, that is better.